pkgs/top-level/python-packages.nix

```diff
@@ -14066,11 +14066,11 @@ in modules // {
   };
 
   mock = buildPythonPackage (rec {
-    name = "mock-1.3.0";
+    name = "mock-2.0.0";
```

Comment:
Do you know how to test this? I run

```
nox-review pr 19701
```

but it fails for all kinds of unrelated reasons. E.g. 

```
error: build of ‘/nix/store/z7gskp44ag26ydi2ydyd29ng2vydkhz9-android-sdk-25.2.2.drv’ failed
```